cmd/testworkflow-toolkit/commands/clone.go

```diff
@@ -95,10 +96,11 @@ func NewCloneCmd() *cobra.Command {
 
 			// Keep the files in temporary directory
 			outputPath := filepath.Join(constants.DefaultTmpDirPath, "repo")
-
 			// Mark directory as safe
 			configArgs := []string{"-c", fmt.Sprintf("safe.directory=%s", outputPath), "-c", "advice.detachedHead=false"}
 
+			fmt.Printf("📦 ")
```

Comment:
Should it be there? Anything can be returned afterwards from the Git or internal error